source/DSCResources/DscPullServerSql/DscPullServerSql.schema.psm1

```
@@ -13,11 +13,11 @@ configuration DscPullServerSql
         [string]
         $RegistrationKey,
 
-        [Parameter(Mandatory = $true)]
+        [Parameter()]
```

PSScriptAnalyzer (1.24.0)

[error] Unexpected attribute 'Parameter'.

(UnexpectedAttribute)